### PR TITLE
change decay to D2KK instead of D2Kpi

### PR DESCRIPTION
**Pull request by andriiusachov**

### CODE REVIEW

File: second-analysis-steps/building-decays-part2.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ d0_sel = SimpleSelection(
     'Sel_D0',
     ConfigurableGenerators.CombineParticles,
     [Pions, Kaons],
```
Comment (vlisovsk) [inline second-analysis-steps/building-decays-part2.md]: You don't need pions here

Comment (andriiusachov) [reply second-analysis-steps/building-decays-part2.md]: Thanks! done

File: second-analysis-steps/building-decays-part2.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ While `SimpleSelection` will allow us to do anything we would do with `Selection
     d0_sel = CombineSelection(
         'Sel_D0',
         [Pions, Kaons],
```
Comment (chrisburr) [inline second-analysis-steps/building-decays-part2.md]: And here

**Comment by chrisburr** [comment]
@andriiusachov Thanks for all of this work 👍

I noticed you were missing from the LHCb team on GitHub, you should now have an invite so you can review and merge other people's PRs if you want.

**Pull request merged**